be/src/storage/schema_change.cpp

```diff
@@ -1277,6 +1283,15 @@ bool SchemaChangeWithSorting::process(RowsetReaderSharedPtr rowset_reader, Rowse
             LOG(WARNING) << "failed to allocate RowBlock.";
             result = false;
             goto SORTING_PROCESS_ERR;
+        } else {
+            // check allocator has enougth memory to do row-block sort
```

Comment:
Better to write more comments.
You should describe what will happen, when there isn't enougth memory.

Reply:
add some comments in https://github.com/StarRocks/starrocks/pull/225/commits/31336bd12aa7acfff082f6ccf48f353010fed8be, review again please